main.go

```diff
@@ -1,5 +1,15 @@
-// servedir command start http server, serving files in current directory
-//       go get foxygo.at/servedir
+// Command servedir starts http server serving files from given
```

Comment:
The servedir command starts an HTTP server, serving files from either the current directory or a specified directory, on the next free ephemeral port or a specified port.

Reply:
Thank you, done.